index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/reset-css@5.0.2/reset.min.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>ProtoGoogle25</title>
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="header-menu__container">
```

Comment:
```suggestion
      <div class="header__container">
```

В данном случае контейнер для компонентов меню является элементом блока header